layouts/partials/hooks/body-end.html

```diff
@@ -6,29 +6,54 @@
 {{ end }}
 {{/* copy-and-paste helper for codenew shortcode */}}
 {{- if or (.HasShortcode "code_sample") (.HasShortcode "code") (.HasShortcode "codenew") -}}
+  {{- $toastrJs := resources.Get "js/toastr-2.1.4.min.js" | minify | fingerprint -}}
```

Comment:
BTW: we'd typically only minify the code in production builds.

Reply:
I don't think the double minification is a big problem. Anyway, OK to remove the `minify` here.